lib/mix/lib/mix/tasks/app.tree.ex

```diff
@@ -20,6 +20,8 @@ defmodule Mix.Tasks.App.Tree do
     * `--pretty` - use Unicode codepoints for formatting the tree.
       Defaults to true except on Windows.
 
+    * `--dot` - produces a DOT graph description of the application tree.
```

Comment:
I'm wondering if we should merge `--dot` and `--pretty` options to one, say `format`?